pkgs/development/coq-modules/mathcomp/default.nix

```diff
@@ -76,7 +76,7 @@ let
     };
     # versions of coq compatible with released mathcomp versions
     coq-versions     = {
-      "1.11.0"       = flip elem [ "8.7" "8.8" "8.9" "8.10" "8.11" ];
+      "1.11.0"       = flip elem [ "8.7" "8.8" "8.9" "8.10" "8.11" "8.12" ];
       "1.11+beta1"   = flip elem [ "8.7" "8.8" "8.9" "8.10" "8.11" ];
```

Comment:
I believe that 1.11+beta1 is compatible with Coq 8.12 as well. But do you really want to keep this beta version in the list of available versions? We usually remove the beta versions of Coq for instance...

Reply:
I don’t know and I didn’t check whether the β versions are compatible.

Maybe Cyril has an opinion about this. (ping @CohenCyril)

Reply:
FWIW, I didn't check in Nix, but I did check in Coq's v8.12 CI before 1.11.0 was released.

Reply:
I’ve just added “8.12” to the “1.11+beta1” list.

Whether said list should be kept or not is a different topic, slightly independent from this PR.

Reply:
I approve of this change.